entrypoint.sh

```diff
@@ -19,7 +19,7 @@ then
   shift 1
   # TODO: This is fine for local development but this file will soon need to change to have
```

Comment:
You will be able to remove this line 20 comment as it will now have gunicorn infront of it